src/angular/autocomplete/autocomplete.module.ts

```diff
@@ -6,7 +6,6 @@ import { SbbOptionModule } from '@sbb-esta/angular/core';
 import { SbbCommonModule } from '@sbb-esta/angular/core';
```

Comment:
I just noticed a previous error of mine; The imports of SbbCommonModule and SbbOptionsModule are both from `@sbb-esta/angular/core` and could be merged.